elcid/templates/episode_list.html

```diff
@@ -31,10 +31,14 @@ <h1 >
                   {% endfor %}
                 </ul>
               </div>
+              <a ng-show="currentTag === 'renal'" href="/letters/#{% url 'renal_handover' %}" target="_blank" class="btn btn-secondary pull-left left-offset-10">
```

Comment:
I'm not sure why this has a hard coded href component and a `url` tag ?

Reply:
This is a consequence of the back and forward of the letters plugin, it is a bit weird...

The theory is that we want it rendered within the angular context to give us access to our angular filters e.g. markdown. I'd rather not accidentally have 2 different markdown renderers. Also the copy and paste directive. 

I agree though its a bit awks!